2016-09-02 00:39:33,307 manager INFO 1.1.0
2016-09-02 00:39:33,309 manager INFO Copyright (C) 2003-2010
2016-09-02 00:39:33,310 manager INFO   Noriaki Ando
2016-09-02 00:39:33,311 manager INFO   Intelligent Systems Research Institute, AIST
2016-09-02 00:39:33,311 manager INFO Manager starting.
2016-09-02 00:39:33,312 manager INFO Starting local logging.
2016-09-02 00:39:33,326 manager.namingmanager INFO NameServer connection succeeded: corba/localhost
2016-09-02 00:39:33,384 ManagerServant INFO Master manager not found
2016-09-02 00:39:33,392 manager WARNING Traceback (most recent call last):
  File "/usr/lib/python2.7/dist-packages/OpenRTM_aist/Manager.py", line 1900, in initManagerServant
    reffile = file(self._config.getProperty("manager.refstring_path"),'w')
IOError: [Errno 13] Permission denied: '/var/log/rtcmanager.ref'

2016-09-02 00:39:33,395 LocalServiceAdmin INFO All the local services are enabled.
2016-09-02 00:39:33,439 manager INFO Component type conf file: outTemp.conf loaded.
2016-09-02 00:39:33,468 ec_base INFO setRate(1000000.000000) done
2016-09-02 00:39:33,470 ec_base INFO setRate(1000.000000) done
2016-09-02 00:39:33,476 rtobject INFO 1 execution context was created.
2016-09-02 00:39:33,489 rtobject INFO Initial active configuration set is default-set.
2016-09-02 00:39:35,771 SdoServiceAdmin WARNING No available SDO service in the factory: 
2016-09-02 00:39:35,772 SdoServiceAdmin ERROR type  already exists.
2016-09-02 00:39:35,781 SdoServiceAdmin WARNING No available SDO service in the factory: IDL:OpenRTM/ComponentObserver:1.0
2016-09-02 00:39:35,782 SdoServiceAdmin ERROR type 12849518-0116-4557-aae0-7261e1961fc5 already exists.
2016-09-02 00:39:51,314 rtobject ERROR Traceback (most recent call last):
  File "/usr/lib/python2.7/dist-packages/OpenRTM_aist/RTObject.py", line 1560, in on_execute
    ret = self.onExecute(ec_id)
  File "outTemp.py", line 183, in onExecute
    if(origin_TempIn.isnew()):
NameError: global name 'origin_TempIn' is not defined

2016-09-02 00:41:57,050 ec_worker ERROR State of the RTC is not ACTIVE_STATE.
2016-09-02 00:42:32,414 ec_worker ERROR State of the RTC is not INACTIVE_STATE.
2016-09-02 00:46:35,980 ec_worker ERROR State of the RTC is not INACTIVE_STATE.
2016-09-02 00:56:01,377 ec_worker ERROR State of the RTC is not ACTIVE_STATE.
2016-09-02 00:56:01,384 ec_worker WARNING ExecutionContext is already stopped.
2016-09-02 00:56:01,385 ec_base ERROR Invoking on_shutdown() for each RTC failed.
